pkg/i18n/GetText.go

```diff
@@ -151,7 +173,7 @@ func GetDefaultLanguage() string {
 	if lang, exists := os.LookupEnv("LANG"); exists {
 		l := parseLang(lang)
 		log.Printf("lang = %#v", l)
-		result = fmt.Sprintf("%s-%s", l.lang1, l.lang2)
+		result = fmt.Sprintf("%s-%s", l.lang1, l.lang2, l.lang3)
 	}
 
 	return result
```

Comment:
Cahngeset part 1 (i.e. lines 97 .. 113) is approved.

Changeset part 2. This is parser routine parsing LANG variable:

$ LANG="en-US.UTF-8" ./golden-point-arm

Provide in struct `Lang` as next values:

```
lang1 = "en"
lang2 = "US"
charset = "UTF-8"
```

Changes in this section (i.e. lines 129 .. 176) doing some misunderstanding for me activities. I can not approve this code part because is broke compilation.

Could you please keep unchange second part (lines 129 .. 176) of your changeset?